src/resources/person.py

```diff
@@ -27,45 +28,66 @@ def get(self):
             comYearMIN  Minimum community year of the user (inclusive)
             comYearMAX  Maximum community year of the user (inclusive)
         """
-        params = request.args
-        query = Person.objects()
-        if params.get('fName'):
-            query = query.filter(fName = params['fName'])
-        if params.get('lName'):
-            query = query.filter(lName = params['lName'])
-        if params.get('email'):
-            query = query.filter(email = params['email'])
-        if params.get('comYearMIN'):
-            query = query.filter(comYear__gte = int(params['comYearMIN']))
-        if params.get('comYearMAX'):
-            query = query.filter(comYear__lte = int(params['comYearMAX']))
-        return [person.to_json() for person in query]
+        try:
+            params = request.args
+            query = Person.objects()
+            if params.get('fName'):
+                query = query.filter(fName = params['fName'])
+            if params.get('lName'):
+                query = query.filter(lName = params['lName'])
+            if params.get('email'):
+                query = query.filter(email = params['email'])
+            if params.get('comYearMIN'):
+                query = query.filter(comYear__gte = int(params['comYearMIN']))
+            if params.get('comYearMAX'):
+                query = query.filter(comYear__lte = int(params['comYearMAX']))
+            return {'response': {'ok': 1.0}, 'searched':[person.to_json() for person in query]}
```

Comment:
the `ok` value should be either an integer or boolean - up to you what you think is a better choice